bench/meili/config.json

```diff
@@ -29,9 +29,7 @@
     }
   },
   "skadi": {
-    "actons": [
-      "height"
-    ],
+    "actons": ["height"],
```

Comment:
unrelated change.. actually the key should be `actions`. not your fault probably, but could you change it to `actions`?

Reply:
Sure @nilsnolde

Reply:
actually it seems like the whole `skadi` member is deprecated, it's not being generated in the config anymore.. you can take it out completely or not even change it, up to you I think